models/userDao.js

```diff
@@ -0,0 +1,66 @@
+const appDataSource = require('./appDataSource');
+
+const signUp = async (first_name, last_name, email, password) => {
+  try {
+    const result = await appDataSource.query(
+      `INSERT INTO users(
+        first_name,
+        last_name,
+        email,
+        password,
+        point
+      ) VALUES (?, ?, ?, ?, 100000)
+      `,
+      [first_name, last_name, email, password, email]
+    );
+    return result;
+  } catch (err) {
+    err.message = 'INVALID DATA';
+    err.statusCode = 400;
+    throw err;
+  }
+};
+
+const duplicationEmail = async (email) => {
+  try {
+    const [result] = await appDataSource.query(
+      `SELECT
+      id
+      FROM users
+      WHERE email = ?
+      `,
+      [email]
+    );
+    return result;
+  } catch (err) {
+    err.message = 'INVALID DATA';
+    err.statusCode = 400;
+    throw err;
+  }
+};
+
+const signIn = async (email) => {
```

Comment:
```suggestion
const getUserByEmail = async (email) => {
```
함수의 이름이 signIn인 반면 실질적인 return 값은 signIn이랑은 연관없기 때문에, 바로 위에서 남겨드린 코멘트와 같은 맥락으로 작성되면 좋을 것 같습니다.